optaplanner-examples/src/main/java/org/optaplanner/examples/nurserostering/domain/contract/MinMaxContractLine.java

```diff
@@ -29,6 +29,26 @@
     private int maximumValue;
     private int maximumWeight;
 
+    public boolean isViolated(int count) {
+        if (minimumEnabled && count < minimumValue) {
+            return true;
+        } else if (maximumEnabled && count > maximumValue) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public int getViolationAmount(int count) {
+        if (minimumEnabled && count < minimumValue) {
+            return (minimumValue - count) * minimumWeight;
+        } else if (maximumEnabled && count > maximumValue) {
+            return (count - maximumValue) * maximumWeight;
+        } else {
+            return 0;
+        }
+    }
```

Comment:
Nice to standardize this.
Consider implementing one method with the other, so that the logic is not duplicated.